R/http.R

```diff
@@ -422,7 +419,7 @@ signatureHeaders <- function(authInfo, method, path, file = NULL) {
     signature <- paste(openssl::base64_encode(hmac), "; version=1", sep = "")
   } else if (!is.null(authInfo$private_key)) {
     # the raw content hash is base64 encoded hex values when using private key.
-    md5 <- openssl::base64_encode(md5)
+    md5 <- openssl::base64_encode(fileMD5(file, raw = TRUE))
```

Comment:
If we eventually get to the point where we drop the openssl dependency, this could use the `jsonlite` equivalent (since that will remain).